addons/binding/org.openhab.binding.freemobilesms/pom.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <artifactId>pom</artifactId>
+    <groupId>org.openhab.binding</groupId>
+    <version>2.5.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>org.openhab.binding.freemobilesms</artifactId>
+  <packaging>eclipse-plugin</packaging>
```

Comment:
Please check other addons for the correct file layout. You are still using the old buildsystem. You MUST also move your binding to "/bundles" and not to "/addons".

Reply:
Migrated to bnd